spec/integration/migration_service_spec.rb

```diff
@@ -287,17 +287,17 @@ def setup_validation_time_expectations
 
       contact = user.contacts.find_by(person: Person.by_account_identifier(contact1_diaspora_id))
       expect(contact).not_to be_nil
-      expect(contact.sharing).to be_falsey
+      expect(contact.sharing).to be_truthy
       expect(contact.receiving).to be_falsey
 
       contact = user.contacts.find_by(person: Person.by_account_identifier(contact2_diaspora_id))
       expect(contact).not_to be_nil
-      expect(contact.sharing).to be_falsey
+      expect(contact.sharing).to be_truthy
       expect(contact.receiving).to be_truthy
 
       contact = user.contacts.find_by(person: Person.by_account_identifier(migrated_contact_new_diaspora_id))
       expect(contact).not_to be_nil
-      expect(contact.sharing).to be_falsey
+      expect(contact.sharing).to be_truthy
```

Comment:
Is there a case where this can actually be false? How does the import deals with ignored users for example?

Reply:
I'll check.

Reply:
The attribute should always be true (and always be set in migration.json), a user I don't share with, has no existence in the contacts table (as far as I analyzed this). 
A blocked user, don't alter the contacts entry - he/she can be blocked or unblocked at any time without loosing information. 

More interestingly.. I can't find any place where a blocked entry is added to the export file. @cmrd-senya Can you confirm this? In the  https://github.com/diaspora/diaspora/blob/1dd2382d0336c3514ec21e2c41e8c99f541dbbb5/app/serializers/export/user_serializer.rb file there is no "blocked".. or anything else? 

So if all agree, this PR might be OK and ready to approve, but the blocked mechanics needs a review.

Reply:
> The attribute should always be true (and always be set in migration.json), a user I don't share with, has no existence in the contacts table (as far as I analyzed this).

No, it's not always true, and a user you don't share with has an entry (`sharing: true, recieving: false`) ... I know this is really confusing, and I just noticed that the `ContactSerializer` also messed this up and `following` and `followed` are the wrong way around (but they aren't used for import yet, so it's "fine"?) ... I need to think about how to handle/fix this the best way, and we probably should un-confuse our database schema with these `sharing`/`receiving` stuff which don't work the way one would expect them to work ... In the federation layer this already got cleaned up and works in a non-confusing way, and the `ContactSerializer` tried to copy the federation model but got it wrong because in the database it's just confusing ...

Anyway ... this PR fixes the problem ... for now at least, so it's fine to merge it ... but we need to revisit this topic because the current state is just a big mess.